api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -406,7 +406,7 @@ public void retireDrug_shouldRetireTheGivenDrug() {
 	}
 	
 	/**
-	 * @see ConcepTServiceImpl#unretireDrug(Drug)
+	 * @see ConceptServiceImpl#unretireDrug(Drug)
 	 */
 	@Test
 	public void unretireDrug_shouldMarkDrugAsNotRetired() {
```

Comment:
Tests must bear expected and actual in asserts method, thats one of its feature of a test.

Reply:
if these code wasnot re lated to testing then there is no problem

Reply:
Yeah, the change isn't related to test but just the the Javadoc comments. The typo was causing error to be displayed by IntelliJ